server/src/main/java/org/candlepin/auth/KeycloakAuth.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.auth;
+
+
+import org.jboss.resteasy.spi.HttpRequest;
+import org.json.JSONObject;
```

Comment:
There is no specific reason. I just found it easier to use it this way. Let me know if it has to be changed

Reply:
Unless there's a specific reason mandating we use an external dependency, we should try to avoid adding them. Especially in cases like this where we already have two other means of processing JSON (Jackson and native Java object).